chemprop/cli/hpopt.py

```diff
@@ -247,6 +252,7 @@ def train_model(config, args, train_loader, val_loader, logger):
         strategy=RayDDPStrategy(find_unused_parameters=True),
         callbacks=[RayTrainReportCallback()],
         plugins=[RayLightningEnvironment()],
+        deterministic=deterministic,
```

Comment:
```suggestion
        deterministic=True if args.pytorch_seed else False,
```
You can remove L242-246 by changing this line of code

Reply:
I agree but would check if PyTorch seed is not None as if it is 0 this will evaluate to False.

Reply:
I have implemented this as `deterministic=True if args.pytorch_seed is not None else False` as suggested by you both.

Reply:
Why not
`deterministic=args.pytorch_seed is not None`

Reply:
Thanks for the suggestion. That's cleaner and I have made this change.